rrd/README.txt

```diff
@@ -0,0 +1,9 @@
+This is an empty folder. This will be copied to Nagios Graph location.
+
+If you are migrating an instance of an existing Nagios, Populate this folder with contents from */rrd/ folder from existing Nagios instance.
+
+Please ensure that configdb also is copied to nonstop_extensions/etc/ path.

```

Comment:
Maybe this requires correct path updates here?

Reply:
No. This path is correct.  The path is relative to path where main Nagios is deployed.